<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalogRecording</class>
 <widget class="QWidget" name="AnalogRecording">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Setting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>420</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This section will read the analog input from ADwin and save they into files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="title">
    <string>Analog Input Recording (With Cycle Run)</string>
   </property>
   <widget class="QPushButton" name="Ain_file_path_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>.....</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ain_samlpling_rate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>25</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="Ain_acquire_enable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ain_file_description">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;quot;YYYY&amp;quot; : Year&lt;/p&gt;&lt;p&gt;&amp;quot;MM&amp;quot; : Month&lt;/p&gt;&lt;p&gt;&amp;quot;DD&amp;quot; : Date&lt;/p&gt;&lt;p&gt;&amp;quot;hh&amp;quot; : Hour&lt;/p&gt;&lt;p&gt;&amp;quot;mm&amp;quot; : Minute&lt;/p&gt;&lt;p&gt;&amp;quot;ss&amp;quot; : Second&lt;/p&gt;&lt;p&gt;&amp;quot;[scan]&amp;quot; : Number of Scan&lt;/p&gt;&lt;p&gt;&amp;quot;[run]&amp;quot; : Number of Run&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Scan[scan].txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>File Path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ain_file_path">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>25</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Sampling Rate (us):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Time_Start">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>65</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Time_End">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>65</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>65</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Interval(ms):</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Saved_channels">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>160</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="title">
    <string>Saved Channels</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="RecordSection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>End</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rate(us)</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Record Section</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RecordSection_Add">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RecordSection_Remove">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
